### DISCUSSION TITLE
Unsuspending a user from the database

### DISCUSSION THREAD

**Discussion by scooby359** [DependencyTrack/dependency-track#2931] [Q&A] [answered]
I'd like to be able to unsuspend a user by altering the database, but not sure how.

My aim is that now I have OIDC access set up, I want to suspend the admin user as it's only secured by a password. But, should something ever go wrong with the OIDC integration, I need to be able to unsuspend the admin user through editing the database so that they can get back in to manage the site.

I've tested altering the `SUSPENDED` column in the database to `FALSE`, but still get a 403 error on attempting to log in, and the user management panel still shows suspended. 

Is there something else that needs changing? Does the backend just need time to sync the database changes?

**Comment by nscuro** [comment, marked as answer]
It's possible that the respective user is still present in the application's L2 cache, yes. You can verify this by restarting the instance and retrying.

**Reply by scooby359** [reply]
This worked - restarted the API service, then the user suspension was updated.

Thanks!
